### PR TITLE
Add lazydocker

### PR DESCRIPTION
**Pull request by wteuber**
#### New App Submission

**App name:**
lazydocker

**Repo or homepage link:**
https://github.com/jesseduffield/lazydocker

**Description:**
A simple terminal UI for both docker and docker-compose.

**Why you think it's awesome:**
I tried both dockly (already listed) and lazydocker.
While both work very well, lazydocker is more user-friendly.

### CODE REVIEW

**Comment by jneidel** [comment]
@wteuber Please add it to the main list (readme) instead.

**Comment by wteuber** [comment]
@jneidel ✅ done

File: readme.md [hunk 1]
```diff
@@ -181,6 +181,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 - [lstags](https://github.com/ivanilves/lstags) - Synchronize Docker images across different registries.
 - [pm2](https://pm2.io/runtime/) - Production Process Manager for Node.js.
 - [dockly](https://github.com/lirantal/dockly) - Interactively manage Docker containers.
+- [lazydocker](https://github.com/jesseduffield/lazydocker) - A simple terminal UI for both docker and docker-compose.
```
Comment (jneidel) [inline readme.md]: [As stated in the contribution guidelines](https://github.com/agarrharr/awesome-cli-apps/blob/master/contributing.md), please move your addition to the end of the category.

**Comment by jneidel** [comment]
@wteuber Please don't force push on PR, it removes the history and messes with code comments...
And when the PR gets merged all commits are squashed into one anyway, so there is no reason not to split changes into different commits.

**Pull request merged**